docs/UserGuide.md

```diff
@@ -46,7 +47,11 @@ It is a convenient platform for you to keep track of your lessons and assignment
    * **`prioritize`**`3 priority/HIGH` : Sets a "HIGH" priority tag for the 3rd assignment.
    
    * **`unprioritize`**`3` : Removes the priority tag for the 3rd assignment.
-
+   
+   * **`done`**`3` : Marks the 3rd assignment as done.
+  
+   * **`undone`**`3` : Marks the 3rd assignment as undone.
+   
```

Comment:
Maybe using "Unmarks the 3rd assignment as done" would be more appropriate since undone is not exactly proper English XD But I think the way it is now is fine too.

Reply:
or perhaps marks the 3rd assignment as not done?

Reply:
okay changed to Andrea's suggestion cause I don't think unmark is a proper English word as well according to Google haha

Reply:
Oh when I googled it, Google said unmark is proper English but okie XD